agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/Exporter.java

```diff
@@ -129,20 +99,55 @@
   private static final AttributeKey<String> AI_LOG_ERROR_STACK_KEY =
       AttributeKey.stringKey("applicationinsights.internal.log_error_stack");
 
-  // note: this gets filtered out of user dimensions automatically since it shares official "peer."
-  // prefix
-  // (even though it's not an official semantic convention attribute)
+  private static final AttributeKey<String> AZURE_NAMESPACE =
+      AttributeKey.stringKey("az.namespace");
   private static final AttributeKey<String> AZURE_SDK_PEER_ADDRESS =
       AttributeKey.stringKey("peer.address");
   private static final AttributeKey<String> AZURE_SDK_MESSAGE_BUS_DESTINATION =
       AttributeKey.stringKey("message_bus.destination");
+  private static final AttributeKey<Long> AZURE_SDK_ENQUEUED_TIME =
+      AttributeKey.longKey("x-opt-enqueued-time");
 
   private static final OperationLogger exportingSpanLogger =
       new OperationLogger(Exporter.class, "Exporting span");
 
   private static final OperationLogger parsingHttpUrlLogger =
       new OperationLogger(Exporter.class, "Parsing http.url");
 
+  static {
+    Set<String> dbSystems = new HashSet<>();
+    dbSystems.add("db2");
+    dbSystems.add("derby");
+    dbSystems.add("mariadb");
+    dbSystems.add("mssql");
+    dbSystems.add("mysql");
+    dbSystems.add("oracle");
+    dbSystems.add("postgresql");
+    dbSystems.add("sqlite");
+    dbSystems.add("other_sql");
+    dbSystems.add("hsqldb");
+    dbSystems.add("h2");
+
+    SQL_DB_SYSTEMS = Collections.unmodifiableSet(dbSystems);
+
+    // TODO need to keep this list in sync as new semantic conventions are defined
+    // TODO make this opt-in for javaagent
+    Set<String> standardAttributesPrefix = new HashSet<>();
```

Comment:
same here.. i feel that otel should take care of all the standdard attribute prefixes.

Reply:
unfortunately there are no otel constants we can use for this, at least not yet